blog/_posts/2022-06-15-scala-3.1.3-released.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.3 released!
+---
+
+We are happy to announce the release of Scala 3.1.3! You can read more about all the improvements and fixes in [the full changelog](https://github.com/lampepfl/dotty/releases/tag/3.1.3). We have also prepared a short highlights of the most exciting additions in the new version.
+
+## Highlights of the release
+
```

Comment:
https://github.com/lampepfl/dotty/pull/15275 - porting of scalac from bash to scala - will enable `cs setup` in future to match bash scalac command.

https://github.com/lampepfl/dotty/pull/15355 - windows batch files now match bash script launchers

Reply:
I'm not sure if this is an essential change for most of our users. A lot of users are not aware of existence of launch scripts. Those who use them will also not see immediate improvements, as (if I remember correctly) this is just a new implementation with the same functionality.

The coursier part may be interesting, but it is still not implemented, so I wouldn't put that in the release notes.